network/validator/authorized_sender_validator_test.go

```diff
@@ -80,17 +93,51 @@ func (s *TestAuthorizedSenderValidatorSuite) TestValidatorCallback_UnAuthorizedS
 
 			authorizedSenderValidator := NewAuthorizedSenderValidator(s.log, s.slashingViolationsConsumer, c.GetIdentity)
 
-			msgType, err := authorizedSenderValidator.Validate(pid, c.Message, c.Channel, true)
-			require.ErrorIs(s.T(), err, message.ErrUnauthorizedRole)
-			require.Equal(s.T(), c.MessageStr, msgType)
-
 			validatePubsub := authorizedSenderValidator.PubSubMessageValidator(c.Channel)
 			pubsubResult := validatePubsub(pid, c.Message)
 			require.Equal(s.T(), pubsub.ValidationReject, pubsubResult)
 		})
 	}
 }
 
+// TestValidatorCallback_AuthorizedUnicastOnChannel checks that AuthorizedSenderValidator.Validate does not return an error
+// for messages sent via unicast that are authorized to be sent via unicast.
+func (s *TestAuthorizedSenderValidatorSuite) TestValidatorCallback_AuthorizedUnicastOnChannel() {
+	fmt.Println(len(s.authorizedUnicastOnChannel))
```

Comment:
this still needed? If so, please convert it to a testing log message

Reply:
https://github.com/onflow/flow-go/pull/3378/commits/13f7d44c424d4f9c40ed8ebb0c997c575a8b5a8b